src/main/java/Main.java

```diff
@@ -111,7 +111,7 @@ public static void main(String[] args) throws Exception {
 
             _main(args);
         } catch (UnsupportedClassVersionError e) {
-            System.err.println("Jenkins requires Java7 or later, but you are running "+
+            System.err.println("Jenkins requires Java 8 or later, but you are running "+
```

Comment:
After looking at this some more it seems '_main' is likely throwing this error since the jenkins war releases are now compiled with java 8. This whole 'main' method seems to be out of date with that change.